src/screens/collection/collection.tsx

```diff
@@ -65,6 +74,8 @@ export const Collection = () => {
     </View>
   );
 
+  const keyExtractor = useCallback(item => `${item.address}_${item.id}`, []);
```

Comment:
you can just move it above the component and don't use `useCallback` that use some resources too